src/main/resources/xl-rules.xml

```diff
@@ -12,124 +12,28 @@
 <rules
         xmlns="http://www.xebialabs.com/xl-deploy/xl-rules">
 
-    <rule name="datical.DeployedProject.CREATE" scope="deployed">
+	<rule name="datical.DeployedProject.CREATE" scope="deployed">
         <conditions>
-            <type>datical.DeployedProject</type>
+			<type>datical.DeployedProject</type>
```

Comment:
Indent?

Reply:
Not sure why identation shows up. I'll fix it.